sources/T-REC-J.1108-201901-P.MSW-E.adoc

```diff
@@ -45,21 +46,26 @@ The most important part of the RoIP transmission system is having the correct ne
 * [[[x743, ITU-T X.743]]] ITU-T Recommendation X.743(1998/06), _Time Management Function_
 
 
+[[definitions]]
```

Comment:
@anermina we probably don't need an anchor for `definitions` because it is normally not referred to.